pkg/service/record/record.go

```diff
@@ -72,13 +71,14 @@ func (r *Recorder) Start(ctx context.Context) error {
 	defer func() {
 		select {
 		case <-ctx.Done():
-			r.telemetry.RecordedTestSuite(newTestSetID, testCount, mockCountMap)
+			break
```

Comment:
This break is not required as the select statement is not in a for loop.

Reply:
Can I open another PR for this issue and close this one? It looks like my local copy became a bit of a mess. I really apologize for that.